modules/extensions-support/src/main/java/org/mule/module/extension/internal/runtime/DefaultExecutionMediator.java

```diff
@@ -90,29 +103,54 @@ public Object execute(OperationExecutor executor, OperationContext context) thro
         }
     }
 
-    private MutableConfigurationStats getMutableConfigurationStats(OperationContext context)
+    private Object executeWithRetryPolicy(OperationExecutor executor, OperationContext context, List<Interceptor> interceptors) throws Throwable
     {
-        ConfigurationStats stats = context.getConfiguration().getStatistics();
-        return stats instanceof MutableConfigurationStats
-               ? (MutableConfigurationStats) stats
-               : null;
+        final RetryPolicyTemplate retryPolicyTemplate;
+        final Optional<ConnectionProvider> connectionProvider;
+
+        connectionProvider = context.getConfiguration().getConnectionProvider();
+
+        retryPolicyTemplate = getRetryPolicyTemplateFromConnectionProvider(connectionProvider);
+        LifecycleUtils.initialiseIfNeeded(retryPolicyTemplate, muleContext);
```

Comment:
Yes, that make sense. Only to be sure, an instance of `OperationMessageProcessor` (With their respective `ExecutionMediator` instance) is created, just once, per operation used in the Mule App, no?